doc/users/next_whats_new/set_colors_ticks_and_ticklabels_rcparams.rst

```diff
@@ -0,0 +1,31 @@
+The color of ticks and tick labels can be set independently using rcParams
+--------------------------------------------------------------------------
+
+The color of ticks and tick labels can now be set independently from the
```

Comment:
This sentence essentially only repeats the heading and can thus be left out. The only addition (mentioning of rc file) is also covered below.

Reply:
OK, I hope it is concise enough now!